batchflow/dsindex.py

```diff
@@ -34,12 +39,48 @@ def from_index(cls, *args, **kwargs):
 
     @classmethod
     def concat(cls, *index_list):
-        """ Create index by concatenating other indices. """
+        """Create index by concatenating other indices.
+
+        Parameters
+        ----------
+        index_list : list
+            indices to be concatenated. each item in the list should
+            have method .index, that returns an 1-d array-like structure
+
+        Returns
+        -------
+        DatasetIndex with one common index
```

Comment:
Check numpydoc format: data type and a decsription should be stated on separate lines